Code/GraphMol/DetermineBonds/DetermineBonds.h

```diff
@@ -0,0 +1,28 @@
+//
+//  Copyright (C) 2022 Sreya Gogineni and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#ifndef RD_DETERMINEBONDS_H
+#define RD_DETERMINEBONDS_H
+#include <RDGeneral/export.h>
+#include <GraphMol/RDKitBase.h>
+
+namespace RDKit {
+// ! assigns atomic connectivity to a molecule using atomic coordinates, disregarding pre-existing bonds
+/*!
+    \param mol is the molecule of interest; it must have a 3D conformer
+    \param useHuckel (optional) if this is  \c true, the extended Huckel theorem will be used to determine connectivity rather than the van der Waals method
```

Comment:
Since I did my thesis with the person who developed the extended Hückel method, I'm picky about how this should be written... we can't easily use the actual `ü` character in code, so we should use the usual substitute: `ue`. 

So please use `Hueckel` instead of `Huckel` everywhere (including the docs)